product_review_for_pc_build.md

```diff
@@ -0,0 +1,58 @@
+# The Best Monitor for Programming and Productivity  
```

Comment:
I don't think this is an appropriate title for this article because the article is about a best equipment for overall productivity, not just the best monitor.